stock_request_tier_validation/views/stock_request_order_view.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2019-2020 ForgeFlow S.L. (https://www.forgeflow.com)
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="view_stock_request_order_filter" model="ir.ui.view">
+        <field name="name">stock.request.select - stock_request_tier_validation</field>
+        <field name="model">stock.request</field>
+        <field name="inherit_id" ref="stock_request.stock_request_search" />
+        <field name="arch" type="xml">
+            <search position="inside">
+                <group expand="0" string="Need actions">
+                    <filter
+                        name="needs_review"
+                        string="Needs my Review"
+                        domain="[('reviewer_ids','in',uid), ('state', 'not in', ['done', 'cancel'])]"
+                        help="My Stock Request Orders to review"
+                    />
+                    <filter
+                        name="tier_validated"
+                        string="Validated"
+                        domain="[('validated', '=', True)]"
+                        help="Stock Requests validated and ready to be confirmed"
+                    />
+                </group>
+            </search>
+        </field>
+    </record>
+</odoo>
```

Comment:
This view should be used in the stock request order model. In Runboat, I can only see two filters duplicated for the stock request model, and none for the stock request order model. I am referring to the *Needs My Review* and *Validated* filters.

![image](https://github.com/OCA/stock-logistics-warehouse/assets/123451545/d14fefe1-e09e-4c9c-8a57-02402038d735)